src/features/people/peopleSaga.js

```diff
@@ -14,6 +19,18 @@ function* fetchPopularPeopleHandler() {
     }
 }
 
+function* fetchPeopleHandlerCrewCast({ payload: id }) {
+    try {
+      const people = yield axios.get(
+        `${APIUrl}movie/${id}/credits?api_key=${APIKey}&language=en-US`
```

Comment:
Do pobrania Crew i Cast jest inny endpoint, `/person/{person_id}/movie_credits`. Rozumiem, że chcesz tutaj pobrać filmy, w których wystąpił aktor? Jeśli tak to trzeba użyć w/w endpointu i zmieniłbym nazwę const people np na const movieCastCrew